docs/source/worker.rst

```diff
@@ -0,0 +1,106 @@
+Worker
+======
+
+Dask-CUDA workers extend the standard Dask worker in two ways:
+
+1) Advanced networking configuration
+2) GPU Memory Pool configuration
+
+These configurations can be defined in the single cluster use case with ``LocalCUDACluster`` or passed to workers on the cli with ``dask-cuda-worker``
+
+Single Cluster configuration
+----------------------------
+
+
+Command Line Tool
+-----------------
+
+New configuration options::
+
+    --device-memory-limit
+    --rmm-pool-size
+    --enable-tcp-over-ucx / --disable-tcp-over-ucx
+    -enable-infiniband / --disable-infiniband
+    --enable-nvlink / --disable-nvlink
+    --net-devices
+
+Full details ``dask-cuda-worker`` options
```

Comment:
I think it's irrelevant to paste the full output of `dask-cuda-worker --help` here. We should definitely elaborate on those options that are specific to `dask-cuda-worker` (like the ones above this), but having a full output will also make it harder to keep in sync. My suggestion is to remove this and later elaborate on the important options.

Reply:
👍